include/cutlass/epilogue/collective/sm90_epilogue_tma_warpspecialized.hpp

```diff
@@ -73,7 +74,7 @@ template <
   class CopyOpR2S_
 >
 class CollectiveEpilogue<
-    Sm90TmaWarpSpecialized<StagesC_,StagesD_,FragmentSize_,ReuseSmemC_>,
```

Comment:
The way we have done this is by setting the type of `ElementD = void`. You can see this for the `ElementC` residual load as well. Can we follow the same pattern here for consistency?

Reply:
I find it not easy to make both C and D void because ElementD is used in many places such as construction tensors and calculating tile shapes in the builder.